network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -392,25 +392,25 @@ def get_editorial_content_index(self):
         indexes = BuyersGuideEditorialContentIndexPage.objects.descendant_of(self)
         return indexes.first()
 
-    def get_hero_featured_article(self) -> Optional["BuyersGuideArticlePage"]:
+    def get_hero_featured_page(self) -> Optional["Page"]:
```

Comment:
Hi @tbrlpld I tried to update the function annotations accordingly after reading about them, however, I was hoping that you could double check my updates. 

Thanks!

Reply:
Thanks for the suggestion Tibor! Updated 👍